catalog/transformer/catalog-transformer-geojson-input/src/test/java/ddf/catalog/transformer/input/geojson/GeoJsonExtensibleTest.java

```diff
@@ -481,6 +483,47 @@ public void testBasicMetacardTypeNoMetacardType()
     verifyBasics(metacard);
   }
 
+  @Test
+  public void testTransformNoId() throws IOException, CatalogTransformerException {
```

Comment:
✏️ Test name may be more clear as `testTransformWithoutIdParam` or something along those lines.